docs/docs/references/applications/README.mdx

```diff
@@ -0,0 +1,67 @@
+import ApplicationTypesIntro from '../../fragments/_application_types_intro.mdx';
+
+# 📱 Applications
+
+## Introduction
+
+Each real-world application that requires user authentication using Logto needs to be registered as an application in Logto.
+
+## Application ID
+
+_Application ID_ is a unique auto-generated identifier to identify your application with the Logto application.
+
+## Application Types
+
+We provide three application types to help developers integrate their applications with Logto:
+
+<ApplicationTypesIntro />
+
+## Application Settings
+
+### Application Name
```

Comment:
A little bit confused by the ApplicationId, ApplicationType, and Application Settings categorization. Shouldn't Application Id and Application Name under a same scope?

Reply:
I thought the application ID is an auto-generated key and it's unmutable.
They're also displayed in different parts on the details page.